typescript/sdk/src/deploy-cctp-adapter.ts

```diff
@@ -0,0 +1,127 @@
+import { ethers } from 'ethers';
+import yargs from 'yargs';
+
+import { types, utils } from '@hyperlane-xyz/utils';
+
+import {
+  ChainMap, // ChainName,
+  Chains,
+  MultiProvider,
+  chainMetadata,
+} from '../src';
+
+import { hyperlaneContractAddresses } from './consts/environments';
+import {
+  AdapterType,
+  CctpAdapterConfig,
+  CctpAdapterDeployer,
+} from './middleware/liquidity-layer-v2/CctpAdapterDeployer';
+
+const circleDomainMapping = [
+  { hyperlaneDomain: chainMetadata[Chains.goerli].chainId, circleDomain: 0 },
+  { hyperlaneDomain: chainMetadata[Chains.fuji].chainId, circleDomain: 1 },
+  {
+    hyperlaneDomain: chainMetadata[Chains.arbitrumgoerli].chainId,
+    circleDomain: 3,
+  },
+];
+
+function buildCctpAdapterConfigMap(owner: types.Address) {
```

Comment:
The configMap and deployer should live in two separate files?
in infra/config/environments/ and infra/src/deployment